src/foundry/client-esm/applications/api/application.d.mts

```diff
@@ -597,6 +673,19 @@ declare class ApplicationV2<
    */
   changeTab(tab: string, group: string, options?: ApplicationV2.ChangeTabOptions): void;
 
+  /**
+   * Perform an event in the application life-cycle.
+   * Await an internal life-cycle method defined by the class.
+   * Optionally dispatch an event for any registered listeners.
+   * @param handler - A handler function to call
+   * @param options - Options which configure event handling
+   * @returns A promise which resoles once the handler is complete if async is true
+   */
+  protected _doEvent<HandlerArgs extends Array<any>, Async extends boolean = false>(
```

Comment:
I think this should be `Async extends boolean | undefined = false` because `options` is optional.